### PR TITLE
Migrate to traefik v2

### PR DESCRIPTION
**Pull request by mmatur**
### Description

This PR migrates to traefik v2

### CODE REVIEW

File: stack-proxy.yml [hunk 1]
```diff
@@ -26,38 +26,36 @@ services:
...
       labels:
-        - traefik.port=8080
-        - traefik.frontend.rule=Host:traefik.dogvs.cat
- 
+        traefik.http.routers.api.rule: Host(`traefik.dogvs.cat`)
+        traefik.http.routers.api.entryPoints: websecure
+        traefik.http.routers.api.service: api@internal
+        traefik.http.routers.api.middlewares: auth
+        traefik.http.services.dummy.loadbalancer.server.port: 4242
+        traefik.http.middlewares.auth.basicauth.users: test:$$apr1$$H6uskkkW$$IgXLP6ewTrSuBkTrqE8wj/,test2:$$apr1$$d9hr9HBB$$4HxwgUir3HP4EsggP/QNo0
```
Comment (BretFisher) [inline stack-proxy.yml]: @mmatur What's the passwords for these users? Can you put the accounts in a yaml comment?

**Comment by BretFisher** [comment]
Thanks for the help @mmatur, a few questions:
1. can you reply to the question above about middleware auth passwords?
2. what about stack-proxy.global.yml. Were you able to test the update to that solution?

File: stack-proxy.yml (hunk 1 above)
Comment (mmatur) [reply stack-proxy.yml]: Passwords are `test/test` and `test2/test2`

**Comment by mmatur** [comment]
@BretFisher 

1. Password are `test/test`and `test2/test2`
2. Regarding `stack-proxy.global.yml` I haven't migrate this part because the [clustering is no more available with Traefik v2 ](https://containo.us/blog/back-to-traefik-2-0-the-online-meet-up-4e4b6411fbfa/#about-kv-stores-distributed-let-s-encrypt)

Yes I have tested it locally. I was not able to test it on DigitalOcean

**Pull request merged**